doc/appendix-de-kassensichv/cash-register-integration/cash-register-integration.md

```diff
@@ -1,58 +1,58 @@
-Using a unique identifier in `cbReceiptReference` that was already used with a ´Sign´ call with ´ReceiptCase´ "Start-Transaction" will end up in an exception.
-The up-counting transaction number defined in TR-03153 is responded behind hash-tag in property ´ftReceiptIdentification´ of ´ReceiptResponse´ prefixed by "IT".
+The regular workflow of the fiskaltrust-SecurityMechanism in the German market for actions running for a short period has the same requirements as long-running ones. There has to be a "Start-Transaction" and a "Finish-Transaction" executed against the TSE. In order to speed up these two steps into one call to the 'Sign' method, a special 'ReceiptCaseFlag' is used. Each time this is used in combination with a usual 'ReceiptCase', a "Start-Transaction" is done behind the scenes upfront the final call, using the given 'ReceiptCase'.
+Using a unique identifier in 'cbReceiptReference' that was already used with a 'Sign' call with 'ReceiptCase' "Start-Transaction" will end up in an exception.
+The up-counting transaction number defined in TR-03153 is responded behind the hash-tag in the property 'ftReceiptIdentification' of 'ReceiptResponse', prefixed by "IT".
 
 ### Receipt for special functions
 
 This section describes receipt types used for special functions on the German market and expands the common descriptions.
 
 #### Zero Receipt
 
-In the following chapters, you can find examples of special cases of zero receipts applicable to the German market.
+You can find examples of special cases of zero receipts applicable to the German market in the following chapters.
 
 #### Start Receipt (Initial Receipt)
 
-There is a number of requirements for the implementation of a new, or a replaced German security mechanism (TSE). This kind of receipt can be used once in a lifetime of the queue/scu combination and initializes also the underlying German security mechanism (TSE) for usage.
-This receipt must be archived. On successful operation, a notification to the tax authority is also created, which reports the active usage of the ftCashboxIdentification and the serial number of the German security mechanism (TSE).
+There are many requirements for implementing a new, or a replaced German security mechanism (TSE). This kind of receipt can be used once in a lifetime of the queue/SCU combination and also initialises the underlying German security mechanism (TSE) for usage.
+This receipt must be archived. On successful operation, for customers of the product "Finanzamtmeldung", tax authorities are notified about the active usage of the ftCashboxIdentification and the serial number of the German security mechanism (TSE) automatically behind the scenes.
```

Comment:
tax authorities are notified about the active usage of the ftCashboxIdentification and the serial number of the German security mechanism (TSE) automatically behind the scenes. >>> tax authorities are notified automatically and behind the scenes about the active usage of the ftCashboxIdentification, and the serial number of the German security mechanism (TSE).